src/__tests__/to-be-visible.js

```diff
@@ -242,4 +243,11 @@ describe('.toBeVisible', () => {
       })
     })
   })
+  describe('element not in document', () => {
```

Comment:
a couple of nit-picks:

```suggestion
  })

  describe('when element not in document', () => {
```

Although my real recommendation here would be to put this test case more towards the beginning, as a sibling case of the first `it` block. I feel there's no need for this to be inside a `describe` block of its own.